mrmustard/lab/gates.py

```diff
@@ -933,7 +933,7 @@ class Amplifier(Channel):
     """
 
     is_gaussian = True
-    short_name = "Amp"
+    short_name = "Amp~"
```

Comment:
Just out of curiosity why these changes?

Reply:
Filippo suggested adding tilde to all noise channels, so that P~ is meaningful enough.